cli/cli-utils/src/getConfig.ts

```diff
@@ -13,7 +16,7 @@ export async function getConfig (
     ignoreNonAuthSettingsFromLocal?: boolean
   }
 ): Promise<Config> {
```

Comment:
Is the intention of this function to support async `updateConfig` hooks? At the moment users can provide async implementations. That mostly works because async functions can return promises, and that gets flattened automatically.

If the intention isn't to support async `updateConfig`, I would check the return type to make sure it's not a promise. If it is intentional, could we `await` the `config.hooks.updateConfig` call for correctness?

Reply:
ok, added support for async updateConfig

Reply:
Thanks!